Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Microsoft 365/Extra Themes/PaletteMicrosoft365SilverDarkMode.cs

```diff
@@ -5483,41 +5483,41 @@ public virtual string BaseFontName
         protected override void DefineFonts()
         {
             // Release existing resources
-            _header1ShortFont?.Dispose();
+            _header1ShortFont.Dispose();
```

Comment:
Can all this tidy up style dispose code be moved ito base ?
And why call dispose, as it will be tidied up (Gen 1) when the object is re-allocated later in these style of functions

Reply:
Reverted, re moving up to base, needs to be a new feature request, so that it can be tracked. Looks like a leftover from 4.x